src/client/client4.ts

```diff
@@ -1618,10 +1618,11 @@ export default class Client4 {
     // Post Routes
 
     createPost = async (post: Post) => {
-        this.trackEvent('api', 'api_posts_create', {channel_id: post.channel_id});
+        const analyticsData = {channel_id: post.channel_id, post_id: post.id, user_actual_id: post.user_id, root_id: post.root_id};
```

Comment:
What's the background on `user_actual_id`?

Reply:
I think Chen or @adamjclarkson would be a better candidate for the answer :)

Reply:
Not sure what the background is, but I know `user_id` is the server id and `user_actual_id` is user id.